src/OpenLoco/src/Paint/PaintSurface.cpp

```diff
@@ -826,19 +824,19 @@ namespace OpenLoco::Paint
                 return;
             }
 
-            if ((_F003D3[self.landObjectId] & LandObjectFlags::unk4) != LandObjectFlags::none)
+            if ((ObjectManager::getLandObjectFlagsCache()[self.landObjectId] & LandObjectFlags::unk4) != LandObjectFlags::none)
             {
                 return;
             }
         }
         else
         {
-            if ((_F003D3[self.landObjectId] & LandObjectFlags::unk5) != LandObjectFlags::none)
+            if ((ObjectManager::getLandObjectFlagsCache()[self.landObjectId] & LandObjectFlags::unk5) != LandObjectFlags::none)
```

Comment:
Considering we're getting this several times, maybe just introduce a local alias for it?